src/dataflow/utilities/node.ts

```diff
@@ -267,9 +267,9 @@ export interface NodeChannelInfo {
   units: string;
   plug: number;
   value: number;
-  virtual?: boolean;
   name: string;
-  method?: (t: number) => number;
+  virtual?: boolean;
+  virtualValueMethod?: (t: number) => number;
```

Comment:
:+1: to the new name, and ability to duck-type. Unsure if the `virtual` prop is necessary or redundant, but seems fine.